sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/DataFeedLiveTests.cs

```diff
@@ -0,0 +1,897 @@
+        {
+            var metrics = new List<DataFeedMetric>() { new ("cost") };
+            var ingestionStartTime = DateTimeOffset.Parse("2020-08-01T00:00:00Z");
+
+            var granularity = new DataFeedGranularity(DataFeedGranularityType.Daily);
+            var schema = new DataFeedSchema(metrics);
+            var ingestionSettings = new DataFeedIngestionSettings(ingestionStartTime);
+
+            return new DataFeed(name, dataSource, granularity, schema, ingestionSettings);
+        }
+
+        private DataFeed GetDataFeedWithOptionalMembersSet(string name, DataFeedSource dataSource)
+        {
+            var metrics = new List<DataFeedMetric>()
+            {
+                new ("cost") { MetricDisplayName = "costDisplayName", MetricDescription = "costDescription" },
+                new ("revenue") { MetricDisplayName = "revenueDisplayName", MetricDescription = "revenueDescription" }
+            };
+            var dimensionColumns = new List<DataFeedDimension>()
+            {
+                new ("city"),
+                new ("category") { DimensionDisplayName = "categoryDisplayName" }
+            };
+            var ingestionStartTime = DateTimeOffset.Parse("2020-08-01T00:00:00Z");
+
+            var granularity = new DataFeedGranularity(DataFeedGranularityType.Custom) { CustomGranularityValue = 1360 };
+            var schema = new DataFeedSchema(metrics) { DimensionColumns = dimensionColumns, TimestampColumn = "timestamp" };
+            var ingestionSettings = new DataFeedIngestionSettings(ingestionStartTime)
+            {
+                IngestionStartOffset = TimeSpan.FromMinutes(30),
+                IngestionRetryDelay = TimeSpan.FromSeconds(80),
+                StopRetryAfter = TimeSpan.FromMinutes(10),
+                DataSourceRequestConcurrency = 5
+            };
+
+            return new DataFeed(name, dataSource, granularity, schema, ingestionSettings)
+            {
+                Description = "This data feed was created to test the .NET client.",
+                AccessMode = DataFeedAccessMode.Public,
+                ActionLinkTemplate = "https://fakeurl.com/%metric/%datafeed",
+                Administrators = new List<string>() { "fake@admin.com" },
+                Viewers = new List<string>() { "fake@viewer.com" },
+                MissingDataPointFillSettings = new () { FillType = DataFeedMissingDataPointFillType.CustomValue, CustomFillValue = 45.0 }
+            };
+        }
+
+        private void ValidateDataFeedWithMinimumSetup(DataFeed dataFeed, string expectedId, string expectedName)
+        {
+            var ingestionStartTime = DateTimeOffset.Parse("2020-08-01T00:00:00Z");
+
+            Assert.That(dataFeed.Id, Is.EqualTo(expectedId));
+            Assert.That(dataFeed.Name, Is.EqualTo(expectedName));
+            Assert.That(dataFeed.Description, Is.Not.Null.And.Empty);
+            Assert.That(dataFeed.Status, Is.EqualTo(DataFeedStatus.Active));
```

Comment:
I see [Status](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/DataFeed/DataFeed.cs#L80) is nullable . Do we know for sure that in all cases in these tests the service will return the Active status?
I guess, in which cases it won't?

Reply:
When a Data Feed is created, its status is `Active`, and this behavior can't be changed during creation, since [the swagger defines it as readonly](https://github.com/Azure/azure-rest-api-specs/blob/3cbc984fcf0fab278b9c28175319f65db1b9162a/specification/cognitiveservices/data-plane/MetricsAdvisor/preview/v1.0/MetricsAdvisor.json#L4442-L4451).

We can, however, update the status to `Paused` during an `Update` operation (one of my next PRs will include this scenario).

The property is nullable because it doesn't need to be specified during an update.

Reply:
Looks like this is the behavior for the nullable properties.